packages/core/FieldList.h

```diff
@@ -203,7 +203,22 @@ long FieldList<T>::length(void) const
 template<class T>
 const Field* FieldList<T>::get(long i) const
 {
-    return reinterpret_cast<const Field*>(&values[i]);
```

Comment:
this code only makes sense when there is a list of fields; I want the code to crash if that is not the case.